.travis.yml

```diff
@@ -30,7 +30,7 @@ install:
   - "time (bash -e .travis/misc-setup.sh | ts)"
   - "cp .travis/localsettings.py localsettings.py"
   - "pip install coverage unittest2 mock --use-mirrors"
-script: "coverage run manage.py test --noinput --failfast --traceback --testrunner=$TEST_RUNNER"
+script: "coverage run manage.py test --noinput --traceback --testrunner=$TEST_RUNNER"
```

Comment:
Yeah I'd revert that. It's nice to see all failing tests when you're doing something that you know is going to break some things; but in steady state yeah it's helpful for failures to short-circuit the rest